job-distributor/node/node.proto

```diff
@@ -4,6 +4,7 @@ package node;
 
 import "google/protobuf/timestamp.proto";
 import "job-distributor/shared/ptypes/label.proto";
+import "orchestrator/proposer/proposer.proto";
```

Comment:
if we import proposer, wouldnt that import everything that is in proposer even when we dont use them all?

Reply:
actually , it is fine haha